log4j-core/src/test/java/org/apache/logging/log4j/core/pattern/ExtendedThrowablePatternConverterTest.java

```diff
@@ -132,6 +132,55 @@ public void testFiltering() {
         assertTrue(result.contains(" suppressed "), "No suppressed lines");
     }
 
+    @Test
+    public void testFiltersStartFrames() {
+        final ExtendedThrowablePatternConverter converter = ExtendedThrowablePatternConverter.newInstance(null, new String[] {
+            "filters.startFrames(org.apache.logging.log4j.core.pattern.ExtendedThrowablePatternConverterTest$ContrivedWebStack.vendorFramework)"});
+
+        Throwable thrown = null;
+        try {
+          new ContrivedWebStack().webContainerEntryPoint();
+        } catch (Throwable t) {
+            thrown = t;
+        }
+
+        final LogEvent event = Log4jLogEvent.newBuilder() //
+                .setLoggerName("testLogger") //
+                .setLoggerFqcn(this.getClass().getName()) //
+                .setLevel(Level.DEBUG) //
+                .setMessage(new SimpleMessage("test exception")) //
+                .setThrown(thrown).build();
+        final StringBuilder sb = new StringBuilder();
+        converter.format(event, sb);
+        final String parentToString = toString(thrown);
+        // System.out.println(parentToString);
+        final String result = sb.toString();
+        // System.out.println(result);
```

Comment:
Would you mind removing these empty comment line starts and commented out lines, please?